tools/wasm-sourcemap.py

```diff
@@ -279,7 +279,11 @@ def build_sourcemap(entries, code_section_offset, prefixes, collect_sources, bas
     if prefixes.provided():
       source_name = prefixes.sources.resolve(file_name)
     else:
-      file_name = os.path.relpath(os.path.abspath(file_name), base_path)
+      file_name = os.path.abspath(file_name)
+      try:
+        file_name = os.path.relpath(file_name, base_path)
+      except Exception:
+        pass      
```

Comment:
can you put the `file_name = os.path.abspath(file_name)` here in the exception block?   Perhaps with a comment `relpath can throw ValueError on windows if paths are not on the same drive`.

Also can you catch just `ValueError`?

Reply:
Sounds sensible, I'm on it.

Regarding verifying, it seems to not reliably work. Of course the local web server of course fails to access files that are outside of its root folder. The net win of this PR is that the build does not abort anymore.
Obviously you still have to keep all mapped files inside the web servers reach.